generators/app/templates/src/app/screens/Dashboard/screens/ListExample/reducer.ts

```diff
@@ -0,0 +1,42 @@
+export interface HomeState {
```

Comment:
If you want to keep this as an example of reducer/state for the screen fine, but change the name so it's called ListExampleState or something like that and use it, otherwise it's an unused file.

Reply:
Yeah I'll remove this files